git/commands.md

```diff
@@ -20,6 +20,7 @@ After running the command respective git's technical files would be created.
 Before starting work, you need to make some settings:
 
 ```
+
 git config --global user.name "Your Name" # specify the name with which the commits will be signed
```

Comment:
Please use the common style of the document. If you are to state optional parameters use <parameter> notation